ci/pipeline.yml

```diff
@@ -120,6 +123,9 @@ jobs:
         ENV: staging
         DNS_ROOT_DOMAIN: ((staging-dns-root-domain))
         DATABASE_ENCRYPTION_KEY: ((staging-database-encryption-key))
+        AWS_GOVCLOUD_REGION: ((staging-aws-govcloud-region))
+        AWS_GOVCLOUD_ACCESS_KEY_ID: ((staging-aws-govcloud-access-key-id))
+        AWS_GOVCLOUD_SECRET_ACCESS_KEY: ((staging-aws-govcloud-secret-access-key))
```

Comment:
Is there a need for `production` versions of these environment variables, or not yet because this is still being tested?

Reply:
not yet - we haven't deployed this to prod yet